src/platform/online/codeforces.py

```
@@ -51,21 +51,32 @@ class Codeforces(CompetitivePlatform):
             url += f"?{payload}"
-        json_data = fetch_url_json(url, throw=False)
-
-        if isinstance(json_data, int) or json_data['status'] != "OK":
-            if isinstance(json_data, int) and json_data != 400:
-                return -1
-            return 0
+        return url
 
+    @classmethod
+    def _api(cls, api: str, **kwargs) -> dict:
+        """传递参数构造 payload，添加首尾下划线可避免与关键词冲突"""
```

Ruff (0.13.1)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)